testsuite/generate-format/tests/detect_format_change.rs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use generate_format::{add_proptest_serialization_tracing, FILE_PATH};
+use serde_reflection::{RegistryOwned, Tracer};
+use serde_yaml;
+use std::collections::BTreeMap;
+
+static MESSAGE: &str = "Run `cargo run -p generate-format -- --record` to refresh the records and `git diff` to study the changes before amending your commit.";
```

Comment:
can we make this prettier?
https://stackoverflow.com/questions/29483365/what-is-the-syntax-for-a-multiline-string-literal

Reply:
Might also want to call out that if you modify this you should list your PR as breaking change